src/test/libslic3r/test_amf.cpp

```diff
@@ -0,0 +1,28 @@
+#include <catch.hpp>
+#include <test_options.hpp>
+#include "Model.hpp"
+#include "IO.hpp"
+
+
+using namespace Slic3r;
+
+SCENARIO("Reading AMF file") {
+    GIVEN("badly formed AMF file (missing vertices)") {
+        auto model {new Slic3r::Model()};
+        WHEN("AMF model is read") {
+            auto ret = Slic3r::IO::AMF::read(testfile("test_amf/5061-malicious.xml"),model);
+            THEN("read should return True") {
```

Comment:
Why should this return true and not false?

Reply:
I strip the illegal facets instead of erroring out.

Reply:
I see. OK